src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -467,79 +439,76 @@ private boolean initializeAppTypeIndex() {
         return fieldMapping;
     }
 
-    private String getIndexName( UUID securableObjectId ) {
-        return SECURABLE_OBJECT_INDEX_PREFIX + securableObjectId.toString();
+    private String getIndexName( UUID entityTypeId ) {
+        return DATA_INDEX_PREFIX + entityTypeId.toString();
     }
 
-    private String getTypeName( UUID securableObjectId ) {
-        return SECURABLE_OBJECT_TYPE_PREFIX + securableObjectId.toString();
+    private String getTypeName( UUID entityTypeId ) {
+        return DATA_TYPE_PREFIX + entityTypeId.toString();
     }
 
-    @Override
-    public boolean createSecurableObjectIndex(
-            UUID entitySetId,
-            List<PropertyType> propertyTypes,
-            Optional<Set<UUID>> linkedEntitySetIds ) {
+    /*** ENTITY DATA INDEX CREATE / DELETE / UPDATE ***/
+
+    public boolean createEntityTypeDataIndex( UUID entityTypeId, List<PropertyType> propertyTypes ) {
         if ( !verifyElasticsearchConnection() ) { return false; }
 
-        String indexName = getIndexName( entitySetId );
-        String typeName = getTypeName( entitySetId );
+        String indexName = getIndexName( entityTypeId );
+        String typeName = getTypeName( entityTypeId );
 
         boolean exists = client.admin().indices()
                 .prepareExists( indexName ).execute().actionGet().isExists();
         if ( exists ) {
             return true;
         }
 
-        final Map<String, Object> securableObjectMapping = prepareSecurableObjectProperties(
+        final Map<String, Object> entityTypeMapping = prepareEntityTypeDataMappings(
                 typeName,
                 propertyTypes,
-                linkedEntitySetIds );
+                Optional.empty() );
 
         try {
             client.admin().indices().prepareCreate( indexName )
                     .setSettings( getMetaphoneSettings() )
-                    .addMapping( typeName, securableObjectMapping )
+                    .addMapping( typeName, entityTypeMapping )
                     .execute().actionGet();
         } catch ( IOException e ) {
-            logger.debug( "unable to create securable object index" );
+            logger.debug( "unable to create entity type data index for {}", entityTypeId );
         }
         return true;
     }
 
-    private boolean addMappingToSecurableObjectIndex(
-            UUID entitySetId,
-            List<PropertyType> propertyTypes,
-            Optional<Set<UUID>> linkedEntitySetIds ) {
```

Comment:
I think we need linkedEntitySetIds here too, because of linked document mappings: 
<entity_set_id>.<property_type_id>

Reply:
all documents are mapped like this now, so those mappings will be created when the entity sets are created